[2025-03-03T02:31:34.610+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-03T02:31:34.630+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v1.second_task scheduled__2025-03-01T00:00:00+00:00 [queued]>
[2025-03-03T02:31:34.641+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v1.second_task scheduled__2025-03-01T00:00:00+00:00 [queued]>
[2025-03-03T02:31:34.641+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-03-03T02:31:34.664+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): second_task> on 2025-03-01 00:00:00+00:00
[2025-03-03T02:31:34.673+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v1', 'second_task', 'scheduled__2025-03-01T00:00:00+00:00', '--job-id', '602', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpbf6m145x']
[2025-03-03T02:31:34.674+0000] {standard_task_runner.py:105} INFO - Job 602: Subtask second_task
[2025-03-03T02:31:34.675+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=665290) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-03T02:31:34.675+0000] {standard_task_runner.py:72} INFO - Started process 665355 to run task
[2025-03-03T02:31:34.728+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_v1.second_task scheduled__2025-03-01T00:00:00+00:00 [running]> on host 25887b48f6fb
[2025-03-03T02:31:34.821+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='our_first_dag_v1' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-01T00:00:00+00:00'
[2025-03-03T02:31:34.823+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-03T02:31:34.823+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-03T02:31:34.823+0000] {logging_mixin.py:190} INFO - Current task name:second_task state:running start_date:2025-03-03 02:31:34.631118+00:00
[2025-03-03T02:31:34.824+0000] {logging_mixin.py:190} INFO - Dag name:our_first_dag_v1 and current dag run status:running
[2025-03-03T02:31:34.824+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-03T02:31:34.825+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-03T02:31:34.826+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2025-03-03T02:31:34.841+0000] {subprocess.py:99} INFO - Output:
[2025-03-03T02:31:34.843+0000] {subprocess.py:106} INFO - hey, I am task2 and will be running after task1!
[2025-03-03T02:31:34.843+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-03T02:31:34.876+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-03T02:31:34.876+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v1, task_id=second_task, run_id=scheduled__2025-03-01T00:00:00+00:00, execution_date=20250301T000000, start_date=20250303T023134, end_date=20250303T023134
[2025-03-03T02:31:34.930+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-03T02:31:34.930+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-03T02:31:34.930+0000] {logging_mixin.py:190} INFO - Dag name:our_first_dag_v1 queued_at:2025-03-03 02:31:31.243951+00:00
[2025-03-03T02:31:34.931+0000] {logging_mixin.py:190} INFO - Task hostname:25887b48f6fb operator:BashOperator
[2025-03-03T02:31:34.969+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-03T02:31:34.989+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-03T02:31:35.003+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-03-04T05:11:19.518+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-04T05:11:19.538+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v1.second_task scheduled__2025-03-01T00:00:00+00:00 [queued]>
[2025-03-04T05:11:19.549+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v1.second_task scheduled__2025-03-01T00:00:00+00:00 [queued]>
[2025-03-04T05:11:19.549+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-03-04T05:11:19.574+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): second_task> on 2025-03-01 00:00:00+00:00
[2025-03-04T05:11:19.589+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v1', 'second_task', 'scheduled__2025-03-01T00:00:00+00:00', '--job-id', '20', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpl7h68tgu']
[2025-03-04T05:11:19.592+0000] {standard_task_runner.py:105} INFO - Job 20: Subtask second_task
[2025-03-04T05:11:19.594+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=259) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-04T05:11:19.595+0000] {standard_task_runner.py:72} INFO - Started process 275 to run task
[2025-03-04T05:11:19.645+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_v1.second_task scheduled__2025-03-01T00:00:00+00:00 [running]> on host e52f68431fc8
[2025-03-04T05:11:19.732+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='our_first_dag_v1' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-01T00:00:00+00:00'
[2025-03-04T05:11:19.732+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-04T05:11:19.746+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-04T05:11:19.746+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2025-03-04T05:11:19.757+0000] {subprocess.py:99} INFO - Output:
[2025-03-04T05:11:19.759+0000] {subprocess.py:106} INFO - hey, I am task2 and will be running after task1!
[2025-03-04T05:11:19.760+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-04T05:11:19.780+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-04T05:11:19.780+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v1, task_id=second_task, run_id=scheduled__2025-03-01T00:00:00+00:00, execution_date=20250301T000000, start_date=20250304T051119, end_date=20250304T051119
[2025-03-04T05:11:19.850+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-04T05:11:19.866+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-04T05:11:19.868+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-03-07T10:44:08.335+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v1.second_task scheduled__2025-03-01T00:00:00+00:00 [queued]>
[2025-03-07T10:44:08.348+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v1.second_task scheduled__2025-03-01T00:00:00+00:00 [queued]>
[2025-03-07T10:44:08.348+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2025-03-07T10:44:08.371+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): second_task> on 2025-03-01 00:00:00+00:00
[2025-03-07T10:44:08.378+0000] {standard_task_runner.py:57} INFO - Started process 369 to run task
[2025-03-07T10:44:08.382+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v1', 'second_task', 'scheduled__2025-03-01T00:00:00+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmppi8brn8c']
[2025-03-07T10:44:08.385+0000] {standard_task_runner.py:85} INFO - Job 28: Subtask second_task
[2025-03-07T10:44:08.432+0000] {task_command.py:416} INFO - Running <TaskInstance: our_first_dag_v1.second_task scheduled__2025-03-01T00:00:00+00:00 [running]> on host 93934f8e361f
[2025-03-07T10:44:08.509+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='our_first_dag_v1' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-01T00:00:00+00:00'
[2025-03-07T10:44:08.511+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-03-07T10:44:08.511+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2025-03-07T10:44:08.525+0000] {subprocess.py:86} INFO - Output:
[2025-03-07T10:44:08.527+0000] {subprocess.py:93} INFO - hey, I am task2 and will be running after task1!
[2025-03-07T10:44:08.527+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-03-07T10:44:08.558+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v1, task_id=second_task, execution_date=20250301T000000, start_date=20250307T104408, end_date=20250307T104408
[2025-03-07T10:44:08.594+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-03-07T10:44:08.647+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
